### ISSUE TITLE
Can 3d room layout be reconstructed with multiple panoramas?

### ISSUE THREAD

**Issue by zhijie-gao** [zouchuhang/LayoutNet#1]

**Comment by zouchuhang** [comment]
Yes, multiple panos provide more evidence for room shape with this system. There is also spherical SfM, MVS, etc

**Comment by alexcolburn** [comment]
@zhijie-gao 
A good resource for Spherical SfM is Pierre Moulon's OpenMVG
https://github.com/openMVG/openMVG/blob/master/docs/sphinx/rst/software/SfM/SfM.rst#notes-about-spherical-sfm

-Alex

**Comment by Wap8CN** [comment]
Can 3d room layout be reconstructed with multiple RGB Images of one room from different position? @zouchuhang
